.github/workflows/pr12-lifecycle-retry-fix.yml

```
@@ -0,0 +1,48 @@
+name: Apply PR 12 lifecycle retry fix
+
+on:
+  push:
+    branches: [codex/close-runtime-lifecycle-boundaries]
+
+permissions:
+  contents: write
+
+jobs:
+  apply:
+    if: startsWith(github.event.head_commit.message, 'chore: apply PR 12 lifecycle retry fix')
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)